examples/bucket_setup_using_autokey/outputs.tf

```
@@ -14,17 +14,12 @@
  * limitations under the License.
  */
 
-output "autokey_config_id" {
+output "bucket_keyhandle" {
   description = "An Autokey configuration identifier."
```

Need to update this description:

```suggestion
output "bucket_keyhandle" {
  description = "The bucket Autokey Key Handle."
```

---

done